sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -81,8 +81,8 @@ statement
         rowFormat?  createFileFormat? locationSpec?
         (TBLPROPERTIES tablePropertyList)?
         (AS? query)?                                                   #createHiveTable
-    | CREATE TABLE (IF NOT EXISTS)? target=tableIdentifier
-        LIKE source=tableIdentifier                                    #createTableLike
+    | CREATE EXTERNAL? TABLE (IF NOT EXISTS)? target=tableIdentifier
```

Comment:
since Spark 2.2, we wanna hide the manage/external concept from users. It looks reasonable to add a `LOCATION` statement in `CREATE TABLE LIKE`, but do we really need the `EXTERNAL` keyword? We don't need to be exactly same with hive.